api/extension-caql.md

```diff
@@ -0,0 +1,55 @@
+# Issuing CAQL Queries
+
+The Circonus Analytics Query Language (CAQL) allows the user to issue complex queries against metric data residing in IRONdb.
+The [CAQL Reference](https://login.circonus.com/resources/docs/user/caql_reference.html) provides comprehensive documentation about functionality offered by the language.
+
+
+## Description
+
+### URI
+
+`/extension/lua/caql_v1`
+
+### Method
+
+GET
+
+### Inputs
+
+Parameters can be submitted as query string parameters or as JSON payload.
```

Comment:
Might want to mention that, if provided via query string parameter, the query needs to be URL-encoded.